archeryutils/constants.py

```diff
@@ -1,2 +1,87 @@
 """Constants used in the archeryutils package."""
-YARD_TO_METRE = 0.9144
+
+_CONVERSIONS = {
+    "metre": 1.0,
+    "yard": 0.9144,
+    "cm": 0.01,
+    "inch": 0.0254,
+}
+
+_YARD_ALIASES = {
+    "Yard",
+    "yard",
+    "Yards",
+    "yards",
+    "Y",
+    "y",
+    "Yd",
+    "yd",
+    "Yds",
+    "yds",
+}
+
+_METRE_ALIASES = {
+    "Metre",
+    "metre",
+    "Metres",
+    "metres",
+    "M",
+    "m",
+    "Ms",
+    "ms",
+}
+
+_CM_ALIASES = {
+    "Centimeter",
+    "centimeter",
+    "Centimeters",
+    "centimeters",
+    "CM",
+    "cm",
+    "CMs",
+    "cms",
+}
+
+_INCH_ALIASES = {
+    "Inch",
+    "inch",
+    "Inches",
+    "inches",
+}
+
+_ALIASES = dict(
+    yard = _YARD_ALIASES,
+    metre = _METRE_ALIASES,
+    cm = _CM_ALIASES,
+    inch = _INCH_ALIASES,
+)
```

Comment:
For consistency with above (and preferred approach) use `{...}` rather than `dict(...)`.